msglibrary/src/test/java/ingsw/group1/msglibrary/EmulatorAddressTest.java

```diff
@@ -0,0 +1,56 @@
+package ingsw.group1.msglibrary;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import ingsw.group1.msglibrary.exceptions.InvalidAddressException;
+
+import static junit.framework.TestCase.fail;
+
+/**
+ * Testing Peers with emulator numbers.
+ *
+ * @author Riccardo De Zen
+ */
+@RunWith(Parameterized.class)
+public class EmulatorAddressTest {
+
+    /**
+     * Parameters for the test, in the form of the various possible emulator addresses.
+     */
+    @Parameterized.Parameters(name = "{index}: {0}")
+    public static Object[][] params() {
+        return new Object[][]{
+                {"5554"},
+                {"+15555215554"},
+                {"5556"},
+                {"+15555215556"},
+                {"5558"},
+                {"+15555215558"},
+        };
+    }
+
+    private String addressToTest;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/manual/RicDeZen/ingsw-group1-message-library/pullRequest?prid=4844869)